networking/podenv.go

```diff
@@ -154,6 +173,25 @@ func (e *podEnv) podNSPathSave() error {
 	return nil
 }
 
+func (e *podEnv) podNSDestroy() error {
+	if e.podNS == nil {
+		return nil
+	}
+	_ = e.podNS.Close() // Don't care if err here
```

Comment:
There's nothing we can do. Most likely error is that it's already closed somehow, but we'd have to do a regex on the error string to tell if this is the case. Either way, we need to unmount.